### DISCUSSION TITLE
Switching to Antenna source

### DISCUSSION THREAD

**Discussion by FalkoSc** [supersaiyanmode/PyWebOSTV#75] [Q&A]
Hi,
the sources can be accessed by: 
`source_control = SourceControl(client)`
How can I switch to source antennaTV
`tv_control = TvControl(client)`

I would like to turn the tv on, thats no problem, and then switch to TV. no matter what source was active before. 

I cannot find the command for that.

**Comment by supersaiyanmode** [comment]
I don't have an Antenna TV so can't test, sorry. What is being returned when you say `source_control.list_sources()`?

**Reply by FalkoSc** [reply]
does not matter if there is a signal or not. I am not able to switch to TV source, I guess theres is no command for it?

only the three hdmi inputs:

```
source_control = SourceControl(client)
sources = source_control.list_sources()

print(sources)
```


_[<InputSource 'HDMI 1'>, <InputSource 'HDMI 2'>, <InputSource 'HDMI 3'>]_

**Reply by supersaiyanmode** [reply]
Since I don't have an antenna input in my TV, I am unable to confirm if it shows up in `.list_sources()`. Can you check `.list_apps()` (You'd use ApplicationControl)? Does it show up there?
